src/features/fireWeather/FireWeatherPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { initOptions } from 'utils/keycloak.json'
+import { selectAuthenticationReducer } from 'app/rootReducer'
+import {
+  createAuthentication,
+  resetAuthentication
+} from './slices/authenticationSlice'
+
+export const FireWeatherPage = () => {
+  const { isAuthenticated } = useSelector(selectAuthenticationReducer)
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    const script = document.createElement('script')
+
+    script.src = `${initOptions.url}/js/keycloak.js`
+    script.async = true
+
+    const keyCloakScript = document.body.appendChild(script)
+
+    keyCloakScript.onload = () => {
+      const keycloak = window.Keycloak(initOptions)
+      keycloak
+        .init({
+          onLoad: 'login-required',
+          promiseType: 'native',
+          checkLoginIframe: false
+        })
+        .then(authenticated => {
+          if (!authenticated) {
+            dispatch(resetAuthentication())
+            window.location.reload()
+          } else {
+            dispatch(createAuthentication(authenticated))
+          }
+          setTimeout(() => {
+            keycloak.updateToken(60)
+          }, 6000)
+        })
+    }
+  }, [dispatch])
+
+  if (isAuthenticated) return <header>Hello World!!!</header>
+  else return <div />
```

Comment:
Can you render a message showing what state users are in? Rendering an empty <div> tag is not user-friendly.

Reply:
@Kyubinhan I am not routing users to empty page with just <Div/>. This is just on way login flow.

Reply:
I am not talking about routing. I am pointing out that in this if condition there is a change to render an empty div tag.